File: tests/core/core_library/main_test.cs
using System;
using System.IO;

using Xunit;

namespace example_test
{
    public class MyTest
    {
        [Fact]
        public void MyTest2()
        {
            Assert.True("bar" == "bar");
        }
        [Fact]
        public void CheckVersion()
        {
            var version = GetType().Assembly.GetName().Version;
            Assert.Equal(1, version.Major);
            Assert.Equal(0, version.Minor);
        }
    }
}

File: tests/core/fsharp_library/main_test.fs
module example_test

open System
open System.IO
open System.Reflection

open Xunit

[<Fact>]
let ``MyTest`` () = Assert.True("bar" = "bar")

[<Fact>]
let ``CheckVersion`` () =
    let version =
        Assembly.GetExecutingAssembly().GetName().Version

    Assert.Equal(1, version.Major)
    Assert.Equal(0, version.Minor)